src/OpenTelemetry.Extensions.Enrichment/OpenTelemetry.Extensions.Enrichment.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion)</TargetFrameworks>
```

Comment:
These new projects are only targeting `net462` was that intentional? Probably want to make them more interesting :smile:

Feel free to introduce new vars...

`<NetMinimumSupportedVersion>net6.0</NetMinimumSupportedVersion>`
`<NetStandardMinimumSupportedVersion>netstandard2.0</NetStandardMinimumSupportedVersion>`

...and make these targets...

`<TargetFrameworks>$(NetFrameworkMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion);$(NetMinimumSupportedVersion)</TargetFrameworks>`

I would love to see more projects in the repo move to that style. Makes the maintainer's life easier when it comes time to bump stuff!

Reply:
Done. Thank you